packages/create-sitecore-jss/src/templates/angular/.vscode/extensions.json

```diff
@@ -4,13 +4,8 @@
     "ms-azuretools.vscode-docker",

```

Comment:
I am thinking we could also get rid of ms-azuretools.vscode-docker, at least in our templates. This is really specific to how we dev internally.

Reply:
To be honest, I was thinking about that, but decided to leave since I thought that we propose extensions like I added: _import cost_, _git lens_
But as we discussed, in such case - it needs to be removed. Done